README.md

```diff
@@ -52,7 +52,8 @@ The `uno` command uses `conf/env.sh.example` for its default configuration which
 sufficient for most users.
 
 Optionally, you can customize this configuration by creating an `env.sh` file and modifying it for
-your environment:
+your environment. Inside this script the variable `UNO_HOME` defaults to the location of the
+cloned Uno repository. You may override this by defining UNO_HOME yourself, if need be.
```

Comment:
To clarify my comment, I am not against the scripts using `UNO_HOME` internally.  Just concerned about suggesting users can set it in the README.